packages/cli/src/databases/migrations/mysqldb/1681134145998-AddV1BannerSetting.ts

```diff
@@ -0,0 +1,16 @@
+import type { ReversibleMigration, MigrationContext } from '@db/types';
+
+export class AddV1BannerSetting1681134145998 implements ReversibleMigration {
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
+		await queryRunner.query(
```

Comment:
do we really want to setup a migration and stuff on the backend for this? could we not use `localStorage.set` and live with users seeing the banner again if they logout and then log back in?

That way we don't have to worry about this migration lingering around until v2.0 (or whenever we decide to flatten the migrations).

Reply:
I'd advise against introducing local storage as yet another place for storing fragments of state. Using the DB for this is perfectly ok IMO, but I agree that we should aim to do it without DB migrations, as no schema adjustment is needed. Could we not simply treat the _absence_ of the `ui.banners.v1.dismissed` entry as `false`?